ibm_vpd_app.cpp

```diff
@@ -127,11 +127,22 @@ static auto isFruInVpdCache(const std::string& objectPath)
  */
 static auto needsRecollection(const nlohmann::json& js, const string& filePath)
 {
-    if (js["frus"][filePath].at(0).value("pluggableAtStandby", false) ||
-        js["frus"][filePath].at(0).value("concurrentlyMaintainable", false))
+    if (js["frus"][filePath].at(0).value("pluggableAtStandby", false))
```

Comment:
Am not finding the key "pluggableAtStandby" in any of vpd jsons. 
Rather I can see this key "replaceableAtStandby".
Should we replace the former with latter if their definition matches?

Reply:
In that case, please raise a defect, we will take that change through different commit.